ledger/appcow.go

```diff
@@ -490,7 +490,9 @@ func (cb *roundCowState) StatefulEval(params logic.EvalParams, aidx basics.AppIn
 	var cx *logic.EvalContext
 	pass, cx, err = logic.EvalStatefulCx(program, params)
 	if err != nil {
-		return false, transactions.EvalDelta{}, ledgercore.LogicEvalError{Err: err}
+		pc, det := cx.PcDetails()
```

Comment:
This is fine as long as cx is non-nil. The default go pattern is that whenever err is non nil, all the other params are..
So, adding a nil check before working on cx wouldn't be a bad idea.

Reply:
right, added the check.